index.js

```diff
@@ -31,6 +32,22 @@ const argv = require('yargs')
         desc: 'Print an HTML file or URL to PDF',
         builder: {
             ...commonOptions,
+            'emulatemedia': {
+                string: true,
+                default: ''
+            },
+            'injectjs': {
+                string: true,
+                default: ''
+            },
+            'magicformat': {
```

Comment:
Since this option is exclusive with `format`, could we make the syntax for this be `--format=auto` so that it's clear this replaces any value for format?

Reply:
Ok, adapted it. How can we add a comment/info that there is a --format=auto?